sdk/src/wallet/account/operations/reissue.rs

```diff
@@ -84,18 +84,22 @@ where
                 let mut conflicting = false;
                 for (index, block_id_) in block_ids.clone().iter().enumerate() {
```

Comment:
That clone seems unnecessary

Reply:
Or maybe not actually, maybe it doesn't complain because of the todos

Reply:
What's with the trailing underscore tho

Reply:
> That clone seems unnecessary

I think it is necessary because we mutate it
> What's with the trailing underscore tho

It's there because the original `block_id` should be returned in case of an error

Reply:
Updated the names